app/code/community/District/Stripe/Model/Method/Cc.php

```diff
@@ -405,7 +405,7 @@ protected function _createCharge(Varien_Object $payment, $amount, $capture = tru
         } catch (Exception $e) {
 
             //Throw payment error
-            throw new Mage_Payment_Model_Info_Exception(Mage::helper('stripe')->__($error['message']));
+            throw new Mage_Payment_Model_Info_Exception(Mage::helper('stripe')->__($e->getMessage()));
```

Comment:
Don't forget to add the `$e` as third parameter ;)

Reply:
Like `throw new Mage_Payment_Model_Info_Exception(Mage::helper('stripe')->__($e->getMessage()), 0, $e);` ?

Reply:
Like `throw new Mage_Payment_Model_Info_Exception(Mage::helper('stripe')->__($e->getMessage()), 0, $e);` ?

Reply:
yep.

Reply:
Done on master